### PR TITLE
Fix typings for Model.columnNameToPropertyName and Model.propertyNameToColumnName static methods

### PR DESCRIPTION
**Pull request by cesumilo**
### Fixed(#1898)

- [x] Add static method type for `Model.columnNameToPropertyName` in `typings/objection/index.d.ts`
- [x] Add static method type for `Model.propertyNameToColumnName` in `typings/objection/index.d.ts`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,8 @@
     "Mikael Lepistö <mikael.lepisto@vincit.com> (https://github.com/elhigu)",
     "Matthew McEachen <matthew-objection@photostructure.com> (https://github.com/mceachen)",
     "Jürg Lehni <juerg@scratchdisk.com> (https://github.com/lehni)",
-    "Igor Savin <kibertoad@gmail.com> (https://github.com/kibertoad)"
+    "Igor Savin <kibertoad@gmail.com> (https://github.com/kibertoad)",
+    "Guillaume Robin<robinguillaume.pro@gmail.com> (https://github.com/cesumilo)"
```
Comment (lehni) [inline package.json:30]: @cesumilo for now this list is quite short and contains only people who wrote rather complex features or fixes or are/were maintainers. Would you be OK if we waited to add you until we're certain we will work together in this way?

Comment (lehni) [reply package.json:30]: Also, since none of the dependencies changed, `package-lock.json` shouldn't actually change?

Comment (cesumilo) [reply package.json:30]: Oh yes sorry 😵‍💫, it was to experiment what it meant for npm. I forgot to remove it.

**Comment by lehni** [comment]
Thanks @cesumilo !

**Comment by lehni** [comment]
Btw, I believe to auto-fix when merging, it would need to say: `Fixes #1898` in the contnet?

**Comment by cesumilo** [comment]
> Btw, I believe to auto-close the issue when merging, it would need to say: `Fixes #1898` in the content?

Good catch! I'll be careful next time 👀

**Comment by lehni** [comment]
… and while we're at it: I prefer to add (#1898) at the end of the commit messages that fixes the issue, not the beginning. It's how we've mostly done it.

**Pull request merged**